### DISCUSSION TITLE
Issue with spacebar while editing rpg members (on Mac)

### DISCUSSION THREAD

**Discussion by mwmayer4** [codefori/vscode-ibmi#1679] [General]
I have Googled for days trying to figure out why when I simply press <space> while editing a rpgle or sqlrpgle source member, a box pops up with a list of items which appear to come from include members used by the current source, fields, and procedures. Almost as if the outline view list of names comes up. Some items in the list I have no idea where they came from. 

So on every line of code I type, I must press the esc key after each time I press the spacebar. It's gotten to the point where the editor is basically not usable.

I have looked in the vs code preferences and turned off anything to do with a "suggestion". I also looked in the JSON settings for code for i.  And the system keyboard shortcuts. Nothing I've tried to change has worked. I am using a Mac.

Any help would be very appreciated!

screen-shot attached. 
![spacebarIssue](https://github.com/codefori/vscode-ibmi/assets/109669620/6397c991-5c9b-4f8f-86b7-5a978aec1bc7)

Thanks!

**Comment by worksofliam** [comment]
You are likely seeing the auto import options. VS Code is making suggestions for system APIs and more that you may want to import automatically (tagged auto-import).

You can turn off the content assist automatically popping up in the VS Code settings.

![image](https://github.com/codefori/vscode-ibmi/assets/3708366/d030621e-3e22-418c-a2d6-ea44bd792354)
